Week1/homework/app.js

```diff
@@ -4,8 +4,136 @@
   const bookTitles = [
     // Replace with your own book titles
     'harry_potter_chamber_secrets',
+    'the_count_of_monte_cristo',
+    'the_alchemist',
+    'footprint',
+    'revitalized_land',
+    'the_man_who_has_two_graves',
+    'home_land',
+    'the_last_theorem_of_fermat',
+    'the_old_man_and_the_sea',
+    'romeo_and_juliet',
   ];
 
   // Replace with your own code
-  console.log(bookTitles);
+  // 1.3
+  function createUlList(element) {
+    const ulList = document.createElement('ul');
+    ulList.setAttribute('id', 'bookTitlesList');
+    ulList.setAttribute('class', 'bookTitlesList');
+    for (let i = 0; i < element.length; i++) {
+      const liList = document.createElement('li');
+      liList.innerText = element[i];
+      liList.setAttribute('class', 'book');
+      ulList.appendChild(liList);
+    }
+    document.body.appendChild(ulList);
+  }
+  //  1.4
+  const bookInformationObject = {
+    harry_potter_chamber_secrets: {
+      title: bookTitles[0],
```

Comment:
In the instructions (1.4) it said that the `title` here be the string you entered in the `bookTitles` array. Your solution is certainly a clever way of solving the homework but I'm not sure it's what Yash intended!